charts/kargo/templates/external-webhooks-server/deployment.yaml

```diff
@@ -82,42 +82,43 @@ spec:
           {{- toYaml . | nindent 8 }}
         {{- end }}
         ports:
-        - containerPort: 9443
-          name: webhooks-server
```

Comment:
This port name wasn't very enlightening as to the protocol used by this port's listener. This is more informative.